pkg/sitewise/api/list_associated_assets.go

```diff
@@ -14,6 +14,33 @@ import (
 
 func ListAssociatedAssets(ctx context.Context, client client.SitewiseClient, query models.ListAssociatedAssetsQuery) (*framer.AssociatedAssets, error) {
 
+	var assetId *string = util.GetAssetId(query.BaseQuery)
+
+	// Recursively load children
+	if query.LoadAllChildren {
```

Comment:
[style] I think it is better to be implemented together with the if statement on line 49, instead of a if check and return here.